lldb/source/Plugins/LanguageRuntime/Swift/SwiftLanguageRuntimeDynamicTypeResolution.cpp

```diff
@@ -2813,6 +2842,12 @@ lldb::addr_t SwiftLanguageRuntimeImpl::FixupAddress(lldb::addr_t addr,
 const swift::reflection::TypeRef *
 SwiftLanguageRuntimeImpl::GetTypeRef(CompilerType type,
                                      TypeSystemSwiftTypeRef *module_holder) {
+  Log *log(GetLog(LLDBLog::Types));
+  if (log && log->GetVerbose())
+    LLDB_LOG(log, "[SwiftLanguageRuntimeImpl::GetTypeRef] Getting typeref for "
```

Comment:
This is (almost) equivalent with
LLDB_LOGV(log, ...)
without the if condition. If you don't mind the __FILE__, __LINE__

Reply:
Personally, I prefer it like this instead of adding `__FILE__` and `__LINE__`